src/tests/Search.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { render, fireEvent } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import '@testing-library/jest-dom/extend-expect';
+import App from '../App';
+import defaultSubreddit from '../helpers/constants';
+
+describe('Search', () => {
+  it('input to be defaultSubreddit when clicking on Search link', () => {
+    const history = createMemoryHistory();
```

Comment:
Usually, I see `MemoryRouter` in tests as it's described in [the docs](https://reacttraining.com/react-router/web/guides/testing/starting-at-specific-routes). Then you can set the `initialEntries` prop to define the page the test starts at